doc/readme/DEVELOPING.md

```diff
@@ -6,7 +6,7 @@ This is a very basic checklist on how to get started with Shogun:
 2. Run the API examples, see [INTERFACES.md](INTERFACES.md), or create your own, see [EXAMPLES.md](EXAMPLES.md)
 4. Run the [tests](#testing).
 
-To avoid spending a lot of our time on explaining the same basic things, please **excessively** use the internet for any questions on the commands and tools needed.
+To avoid spending a lot of our time explaining the same basic things, please **excessively** use the internet for any questions on the commands and tools needed.
```

Comment:
The original seems correct to me "spend time on..."

Reply:
For "spend" verb, if it is used to express activity, we don’t use a preposition, and we put the verb in the –ing form.

Reply:
OK!